examples/tutorial/jupyter/execution/pandas_on_ray/cluster/exercise_5.py

```diff
@@ -0,0 +1,51 @@
+import os
+import time
+import ray
+import modin.pandas as pd
+from modin.utils import execute
+
+ray.init(address="auto")
+cpu_count = ray.cluster_resources()["CPU"]
+assert cpu_count == 576, f"Expected 576 CPUs, but found {cpu_count}"
+
+file_path = "big_yellow.csv"
+file_size = os.path.getsize(file_path)
+
+
+# get human readable file size
+def sizeof_fmt(num, suffix="B"):
+    for unit in ("", "K", "M", "G", "T"):
+        if abs(num) < 1024.0:
+            return f"{num:3.1f}{unit}{suffix}"
+        num /= 1024.0
+    return f"{num:.1f}P{suffix}"
+
+
+print(f"File size is {sizeof_fmt(file_size)}")  # noqa: T201
+
+t0 = time.perf_counter()
+df = pd.read_csv(file_path, quoting=3)
```

Comment:
Why do we not wrap this into execute? I think we should call it for read_csv too.

Reply:
This comment is outdated since we decided not to use "execute" function at all in the tutorial.